db/migrate/20210415203352_create_employees.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.string :name
       t.string :email
       t.string :position
+      t.integer :employee_number
```

Comment:
May I ask, What's the difference between this field and the private number field?